Shared/AppInfo.swift

```diff
@@ -23,37 +23,28 @@ open class AppInfo {
 
     /// Return the shared container identifier (also known as the app group) to be used with for example background
     /// http requests. It is the base bundle identifier with a "group." prefix.
-    open static func sharedContainerIdentifier() -> String {
-        return "group." + appGroupSuffix()
+    open static var sharedContainerIdentifier: String {
+        return "group." + baseBundleIdentifier
     }
 
     /// Return the keychain access group.
     open static func keychainAccessGroupWithPrefix(_ prefix: String) -> String {
-        return prefix + "." + appGroupSuffix()
-    }
-
-    /// Returns the base string used to append group. and the team identifier to
-    /// that is used for the app's entitlements.
-    open static func appGroupSuffix() -> String {
-        return Bundle.main.object(forInfoDictionaryKey: "MozGroupSuffix") as! String
+        return prefix + "." + baseBundleIdentifier
     }
 
     /// Return the base bundle identifier.
     ///
     /// This function is smart enough to find out if it is being called from an extension or the main application. In
     /// case of the former, it will chop off the extension identifier from the bundle since that is a suffix not part
     /// of the *base* bundle identifier.
-    open static func baseBundleIdentifier() -> String? {
+    open static var baseBundleIdentifier: String {
```

Comment:
This doesn't need to be an optional since it should always be there. If it's not something is definitely wrong.